spec/features/work_show_spec.rb

```diff
@@ -70,19 +70,17 @@
     end
 
     it "allows adding work to a collection", clean_repo: true, js: true do
-      skip 'maybe embargogeddon....maybe not?' do
-        click_button "Add to collection" # opens the modal
-        # Really ensure that this Collection model is persisted
-        Collection.all.map(&:destroy!)
-        persisted_collection = FactoryBot.create(:collection_lw, user: user, collection_type: multi_membership_type_1)
-        select_member_of_collection(persisted_collection)
-        click_button 'Save changes'
-
-        # forwards to collection show page
-        expect(page).to have_content persisted_collection.title.first
-        expect(page).to have_content work.title.first
-        expect(page).to have_selector '.alert-success', text: 'Collection was successfully updated.'
-      end
+      click_button "Add to collection" # opens the modal
```

Comment:
this spec is currently failing, but I do not believe it was related to my embargo work (#6098). I haven't been able to narrow down where the error was introduced though to try and be able and figure out, "easier", what broke it.

ref: [commit](https://github.com/samvera/hyrax/commit/58de8c60d9d85bf6392964a087d8717e6bba59df) where I commented these out with hesitation

Reply:
now skipping the spec conditionally, with a reason.

<details>
<summary>dassie</summary>

<img width="721" alt="Screenshot 2023-08-07 at 9 43 11 AM" src="https://github.com/samvera/hyrax/assets/29032869/bd5032aa-1e59-460d-9d14-f953d75d21c6">
</details>

<details>
<summary>koppie</summary>

<img width="720" alt="Screenshot 2023-08-07 at 9 43 32 AM" src="https://github.com/samvera/hyrax/assets/29032869/2357e488-128b-4ec2-9072-0adb6ccb077d">
</details>

Reply:
ref: https://github.com/samvera/hyrax/issues/6135